presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroupManager.java

```diff
@@ -116,16 +120,18 @@ public InternalResourceGroupManager(
             ServerConfig serverConfig)
     {
         requireNonNull(queryManagerConfig, "queryManagerConfig is null");
+        this.queryManagerConfig = requireNonNull(queryManagerConfig, "queryManagerConfig is null");
```

Comment:
nit `requireNonNull(queryManagerConfig, "queryManagerConfig is null")` is repeated twice.
We can get rid of the first one.